README.md

```diff
@@ -1,51 +1,207 @@
+* [Documentation](#documentation)
+* [Requirements and installation](#requirements-and-installation)
+* [Usage example](#usage-example)
+* [Use cases](#use-cases)
+* [Demo app (Docker)](#demo-app)
+* [Suggestions and questions](#suggestions-and-questions)
+
+## Features
+
+This library consists of the APIs interface and a couple of Django goodies, 100% covering [Upload](https://uploadcare.com/api-refs/upload-api/), [REST](https://uploadcare.com/api-refs/rest-api/) and [URL](https://uploadcare.com/api-refs/url-api/) Uploadcare APIs.
+
+* Enable obsessively friendly [file uploads](http://uploadcare.com/products/file-uploader/)
+* Manage stored files RESTfully: from malware detection to file conversion
+* [Optimize and process images on the fly](https://uploadcare.com/cdn/image-cdn/) right before delivery time
+
+<!-- Enable obsessively friendly file uploads:
+* Built-in File Uploader
+* Upload files from anywhere
+* Multipart uploading for large files
+* File size and MIME type filtering
+* Malware protection
+* Signed uploads
+* Uploading network to speed up uploading jobs
+
+Store files the way you need:
+* Define file storing behaviour
+* Create projects with separate settings programmatically
+* Connect custom storage
+
+Handle all files smart & RESTfully:
+* CRUD files and their metadata
+* Scan for unsafe and malicious content
+* Convert documents and other files
+* Encode and transform videos
+* Recognize objects in the image
+* Add arbitrary file metadata
+* Control events with webhooks
+
+Process images on the fly right before delivery time:
+* Compression
+* Geometry
+* Colors
+* Definition
+* Overlays
+* Rotations
+* Recognition
+
+Fast and reliable delivery:
+* Worldwide CDN
+* Custom CDN CNAME
+* Signed URLs
+* Search engine indexing behaviour
+* Proxy to fetch and deliver remote files -->
+
+## Documentation
+
+Detailed specification of this library is available [on RTD](https://pyuploadcare.readthedocs.io/en/latest/).
+
+Please note that this package uses Uploadcare [API keys](https://app.uploadcare.com/projects/-/api-keys) and is intended to be used in server-side code only.
+
+## Requirements and installation
+
+* Python 3.8, 3.9, 3.10, 3.11, 3.12
+
+To use pyuploadcare with Python 3.6 or 3.7 please install `pyuploadcare < 5.0`.
+
+To use pyuploadcare with Python 2.7 please install `pyuploadcare < 3.0`.
+
+Django compatibility:
+
+| Py/Dj | 2.2 | 3.0 | 3.1 | 3.2 | 4.0 | 4.1 | 4.2 | 5.0 |
+| ----- | --- | --- | --- | --- | --- | --- | --- | --- |
+| 3.8   | X   | X   | X   | X   | X   | X   | X   |     |
+| 3.9   | X   | X   | X   | X   | X   | X   | X   |     |
+| 3.10  |     |     |     | X   | X   | X   | X   | X   |
+| 3.11  |     |     |     |     |     | X   | X   | X   |
+| 3.12  |     |     |     |     |     |     | X   | X   |
+
+In order to install `pyuploadcare`, run these commands in CLI:
+
+```bash
+pip install pyuploadcare
+```
+
+or, if you prefer it the old way:
+
+```bash
+easy_install pyuploadcare
```

Comment:
`easy_install` is so dead; I doubt if anyone uses it in 2024 (I know this piece is copied from RTD).